### ISSUE TITLE
eject fails on OSX with unhelpful error message

### ISSUE THREAD

**Issue by fhunleth** [fhunleth/fwup#29]
Here's the error:

 `fwup: eject failed: 0xc010 (49168) (null))`

This was reported by @wsmoak on the elixir-lang slack. I'm not sure if improvements to the error message would help debug this particular issue. Check if error 49168 could be a commonly encountered error if the SDCard is busy and produce a better error message.

**Comment by rkenzhebekov** [comment]
@fhunleth, I got this same error:
```
fwup: eject failed: 0xc010 (49168) (null))
```

**Comment by fhunleth** [comment]
I'm not a Mac person, but if anyone has any clues to this error or what's happening, please share. I also can't reproduce it.

**Comment by fhunleth** [comment]
After some googling, it sounds like people are getting around this error by disabling their anti-virus software. I'm wondering if those of you who've seen this have an anti-virus program running.

**Comment by wsmoak** [comment]
I turned off Bitdefender, but I still get this (with fwup 0.8.0)

```
$ mix firmware.burn
Use 3.71 GiB memory card found at /dev/rdisk2? [Yn] y
100%
fwup: eject failed: 0xc010 (49168) (null))
```

**Comment by fhunleth** [comment]
Bummer. Thanks for letting me know. I guess it's time for more googling. If anyone knows what error 49168 is, that would be helpful. I can't find it.

**Comment by fhunleth** [comment]
Since it seems like this error is harmless, I silenced it in 8dbc0ca01f038123263b8ea8601378d0493b6a07. I never could find information about the error.

**Comment by fhunleth** [comment]
Updates: 

It turns out that 0.15.0 caused the error to hit more often. I found and fixed an issue and released 0.15.1. I re-enabled the error message in 0.15.1 since I thought that this issue was really fixed now. 

Last night I witnessed an eject failure with 0.15.1. The person who had the laptop said that it was in really poor shape and had many issues with USB devices. She was using a USB Multicard reader to program the MicroSD card, so that may be related. Her SDCard slot didn't work. She insisted that the eject problem was likely due to her laptop. However, the error definitely was happening, so reopening...
